utils/seed_asset.py

```diff
@@ -22,6 +22,14 @@
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        prog="seed_asset",
+        description="""
+            The program for importing of assets
```

Comment:
we should include the expected column names here.

Reply:
and file format, etc

Reply:
Okay, Will do that. Thanks